container/test-impl-base/src/main/java/org/jboss/arquillian/container/test/impl/RunModeUtils.java

```diff
@@ -38,7 +40,7 @@
    private RunModeUtils() { }
    
    /**
-    * Check is this should run as client.
+    * Returns if the given test should run as client.
```

Comment:
Fair enough :) I'm not native english speaker anyway so I'm most of the times wrong :D